app/src/main/java/com/example/henzoshimada/feeltrip/MoodAdapter.java

```diff
@@ -76,6 +80,8 @@ public View getView(int position, View convertView, ViewGroup parent) {
         viewHolder.userName.setText(mood.getUsername());
         viewHolder.date.setText(mood.getDate().toString());
         viewHolder.description.setText(mood.getDescription());
+        viewHolder.append.setText(mood.getAppend() + mood.getEmotionalState());
+        viewHolder.emoji.setImageBitmap(null); //TODO: pass the emoji into here
```

Comment:
Append the append from elasticsearch to the end of the mood description. Allows us to do so much more with our UI.